Sources/BrowserServicesKit/SecureVault/AutofillSecureVault.swift

```diff
@@ -18,6 +18,19 @@
 
 import Foundation
 import Common
+import SecureStorage
+
+public typealias AutofillVaultFactory = SecureVaultFactory<DefaultAutofillSecureVault<DefaultAutofillDatabaseProvider>>
+
+public let AutofillSecureVaultFactory: AutofillVaultFactory = SecureVaultFactory<DefaultAutofillSecureVault>(
+    makeCryptoProvider: {
+        return AutofillCryptoProvider()
+    }, makeKeyStoreProvider: {
+        return AutofillKeyStoreProvider()
+    }, makeDatabaseProvider: { key in
+        return try DefaultAutofillDatabaseProvider(key: key)
+    }
+)
```

Comment:
Not totally thrilled to introduce some globals like this, but the callsites in the clients admittedly look nice. Would be open to move this elsewhere.